### DISCUSSION TITLE
Customize Admin/Project Excel Export

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#2809] [Questions & Answers]
Hello,

is there a way to customize the excel export of the projectlist ? 
In need the budget and timebudget in the xls file.

Navigation
Administration -> Projekte -> Download/Export Button (rechts oben)

Thank you very much

**Comment by kevinpapst** [comment]
The configured value can be added, seems I forgot to add it... I will add it for the next release (see here: https://github.com/kevinpapst/kimai2/pull/2812)

The remaining/used budget is not available and will not be added. 
This export is exclusively for the plain values of the filtered entities (activities/projects ...).

**Reply by flexminister** [reply]
Is it generally possible to generate an evaluation for all projects with total and remaining hours?

**Reply by kevinpapst** [reply]
@ibens open new discussions for new questions please. Check "Reports > Project overview" ... maybe that is what you want?
